src/components/letter/LetterDetailSection.tsx

```
@@ -234,18 +232,17 @@ export default function LetterDetailSection({
   }, [setFixedAction, handleSaveCard]);
 
   const displayReceivedAt = (() => {
-    if (receivedAt === null) return "-";
+    if (receivedAt === null) return '-';
     if (!receivedAt) return receivedAt;
-    let value = receivedAt.replace(/\./g, "-");
-    const parts = value.split("-");
+    let value = receivedAt.replace(/\./g, '-');
```

ESLint

[error] 'value' is never reassigned. Use 'const' instead.

(prefer-const)